waveform_editor/gui/globals_dialog.py

```diff
@@ -0,0 +1,32 @@
+import panel as pn
+import param
+from panel.viewable import Viewer
+
+
+class YamlGlobalsDialog(Viewer):
+    visible = param.Boolean(allow_refs=True)
+
+    def __init__(self, config):
+        super().__init__()
+        self.config = config
+        self.modal = pn.Modal(pn.Param(self.config.globals, name="YAML Globals"))
```

Comment:
The GUI element for MD URIs is not very clear:
<img width="365" height="215" alt="image" src="https://github.com/user-attachments/assets/f2bf4c44-2a7e-4d5f-b42d-f8265176db5b" />

Can we use a different widget? Looking at the panel gallery perhaps the [Tabulator](https://panel.holoviz.org/reference/widgets/Tabulator.html) is the easiest way to get a nice UI?